lib/active_admin/views/components/paginated_collection.rb

```diff
@@ -113,10 +113,10 @@ def page_entries_info(options = {})
           if @display_total
             total  = collection.total_count
             I18n.t 'active_admin.pagination.multiple', :model => entries_name, :total => total,
-              :from => offset + 1, :to => offset + collection_size
+              :from => offset + 1, :to => offset + collection_size(collection)
           else
             I18n.t 'active_admin.pagination.multiple_without_total', :model => entries_name,
-              :from => offset + 1, :to => offset + collection_size
+              :from => offset + 1, :to => offset + collection_size(collection)
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.